src/main/java/gov/usgs/wma/waterdata/LoadTimeSeries.java

```diff
@@ -15,6 +15,9 @@
 
 	public static final String STATUS_SUCCESS = "success";
 	public static final String STATUS_FAIL = "fail";
+	public static final String FAIL_MESSAGE_NO_RECORDS = "No records found";
```

Comment:
Bonus points, have the message here match the logged message. String.format() could be used to get the dynamic values into the message. https://www.baeldung.com/string/format